.dockerignore

```diff
@@ -0,0 +1,29 @@
+**/.classpath
```

Comment:
I would prefer to keep the repository reasonably clean. Is this file necessary? As far as I can tell, without the file the build context would just be a bit bigger.